### ISSUE TITLE
"Hide" button on alert banners does not meet colour contrast requirement in hover state

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_base#375]
... I think?

![image](https://user-images.githubusercontent.com/19519457/219038886-6ade7541-edad-4d75-94bd-a314fa499bdb.png)

**Comment by markconroy** [comment]
Do you mean the background colour (white) against the banner background colour (yellow)?

I wonder is that an a11y issue? When hovering, you know where you are since your cursor is moving in that position and it was you who placed it there. It's different for focus, where you may not know where the next focusable element is. In this case, the focus is fine with the outline being set, so it's only the hover state, which is also taken care of by having an underline on hover.

@msayoung any thoughts?

**Comment by msayoung** [comment]
Have you never lost your mouse-cursor @markconroy --- nope, just me then :laughing: 

https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html 

As I understand it, it's not a WCAG non-compliance / failure as the boundary isn't the only way to perceive the button (we have high enough contrast text). However the boundary being high enough contrast is recommended. 


"Boundaries
This success criterion does not require that controls have a visual boundary indicating the hit area, but if the visual indicator of the control is the only way to identify the control, then that indicator must have sufficient contrast. If text (or an icon) within a button or placeholder text inside a text input is visible and there is no visual indication of the hit area then the Success Criterion is passed. If a button with text also has a colored border, since the border does not provide the only indication there is no contrast requirement beyond the text contrast ([1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum)). Note that for people with cognitive disabilities it is recommended to delineate the boundary of controls to aid in the recognition of controls and therefore the completion of activities."

**Comment by markconroy** [comment]
PR ready!
